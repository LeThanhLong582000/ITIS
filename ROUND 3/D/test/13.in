9
201
321
9
1
181
201
573
837
81
