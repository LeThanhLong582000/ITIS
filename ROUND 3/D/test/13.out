1229
2131
23
2
1087
1229
4159
6449
419
